tests/default_vars.sh

```diff
@@ -161,6 +161,42 @@ elif [[ $MACHINE_ID = jet.* ]]; then
   TASKS_stretch=48 ; TPN_stretch=12 ; INPES_stretch=2 ; JNPES_stretch=4
   TASKS_strnest=96 ; TPN_strnest=12 ; INPES_strnest=2 ; JNPES_strnest=4
 
+  TASKS_cpl_dflt=192; TPN_cpl_dflt=24; INPES_cpl_dflt=3; JNPES_cpl_dflt=8
+  THRD_cpl_dflt=1; WPG_cpl_dflt=6;  MPB_cpl_dflt="0 143"; APB_cpl_dflt="0 149"
+  OPB_cpl_dflt="150 179"; IPB_cpl_dflt="180 191"
+
+  TASKS_cpl_dflt_wwav=204; TPN_cpl_dflt_wwav=24; INPES_cpl_dflt_wwav=3; JNPES_cpl_dflt_wwav=8
+  THRD_cpl_dflt_wwav=1; WPG_cpl_dflt_wwav=6;  MPB_cpl_dflt_wwav="0 143"; APB_cpl_dflt_wwav="0 149"
+  OPB_cpl_dflt_wwav="150 179"; IPB_cpl_dflt_wwav="180 191"; WPB_cpl_dflt_wwav="192 203"
+
+  TASKS_cpl_thrd=120; TPN_cpl_thrd=18; INPES_cpl_thrd=3; JNPES_cpl_thrd=4
```

Comment:
How is TPN_cpl_thrd=18 related to TPN_cpl_dflt=24? On a related note, I see that TPN_cpl_dflt=40 and TPN_cpl_thrd=40 for, example, Hera. Is this intended (cf. TPN_dflt=40 and TPN_thrd=20 for Hera)?

Reply:
I had noticed that also in Bin's original PR. They're not consistent across platforms. I had created an issue #320 a while ago as a way of getting around this problem of having to manually edit the numbers for the coupled model.